pkg/taskdir/definitions/definitions.go

```diff
@@ -124,6 +125,14 @@ func (this Definition) GetKindAndOptions() (api.TaskKind, api.KindOptions, error
 		if err := mapstructure.Decode(this.REST, &options); err != nil {
 			return "", api.KindOptions{}, errors.Wrap(err, "decoding REST definition")
 		}
+		// API server expects JSONBody as a string.
+		if _, ok := options["jsonBody"].(string); options["jsonBody"] != nil && !ok {
```

Comment:
minor suggestion, check `!ok` first:

```suggestion
		if _, ok := options["jsonBody"].(string); !ok && options["jsonBody"] != nil {
```